Final/fdlist.h

```diff
@@ -0,0 +1,81 @@
+#ifndef FD_LIST_H
+#define FD_LIST_H
+
+#include <stdio.h>
+#include <stdlib.h>
+
+struct node {
+	int data;
+	struct node *next;
+};
+
+struct node* createFDList(int data) {
+	int size_of_node = sizeof(struct node);
+	struct node *head = (struct node*) malloc(size_of_node);
+	head->data = data;
+	head->next = NULL;
+	return head;
+}
+
+void pushToList(struct node *head_ref, int data) {
+	int size_of_node = sizeof(struct node);
+	struct node *new_node = (struct node*) malloc(size_of_node);
+	new_node->data = data;
+	struct node *temp_node = head_ref->next;
+	head_ref->next = new_node;
+	new_node->next = temp_node;
+	//printf("%i %s\n", data, "added");
+}
+
+int removeFromList(struct node *head_ref, int data) {
+	struct node *pre_node = head_ref;
+	struct node *cur_node = head_ref->next;
```

Comment:
all of your methods assume that the first node in the list is a dummy node. while this works, this is generally bad practice for linked lists. see if you can refactor this code to not rely on this dummy node.